applications/StructuralMechanicsApplication/custom_constitutive/linear_isotropic_damage_plane_strain_2d.cpp

```diff
@@ -0,0 +1,103 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Marcelo Raschi
+
+#include "linear_isotropic_damage_plane_strain_2d.h"
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+LinearIsotropicDamagePlaneStrain2D::LinearIsotropicDamagePlaneStrain2D()
+    : LinearIsotropicDamage3D()
+{
+}
+
+//********************************COPY CONSTRUCTOR************************************
+//************************************************************************************
+LinearIsotropicDamagePlaneStrain2D::LinearIsotropicDamagePlaneStrain2D(
+    const LinearIsotropicDamagePlaneStrain2D &rOther)
+    : LinearIsotropicDamage3D(rOther)
+{
+}
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer LinearIsotropicDamagePlaneStrain2D::Clone() const
+{
+    LinearIsotropicDamagePlaneStrain2D::Pointer pclone(new LinearIsotropicDamagePlaneStrain2D(*this));

```

Comment:
kratos::make_shared

Reply:
True. I'll update the other CLs in an upcoming PR.